docs/user/docker_config.rst

```diff
@@ -0,0 +1,158 @@
+      # Grafana for the live monitor.
+      # --------------------------------------------------------------------------
+      grafana:
+        image: grafana/grafana:5.4.0
+        restart: always
+        ports:
+          - "127.0.0.1:3000:3000"
+        environment:
+          - GF_INSTALL_PLUGINS=grafana-simple-json-datasource, natel-plotly-panel
+        volumes:
+          - /srv/grafana:/var/lib/grafana
+
+      # --------------------------------------------------------------------------
+      # sisock Components
+      # --------------------------------------------------------------------------
+      sisock-crossbar:
+        image: simonsobs/sisock-crossbar:latest
+        ports:
+          - "127.0.0.1:8001:8001" # expose for OCS
+        volumes:
+          - ./.crossbar:/app/.crossbar
+        environment:
+             - PYTHONUNBUFFERED=1
+
+      # --------------------------------------------------------------------------
+      # OCS Agents
+      # --------------------------------------------------------------------------
+      ocs-aggregator:
+        image: simonsobs/ocs-aggregator-agent:latest
+        hostname: ocs-docker
+        user: "9000"
+        volumes:
+          - ${OCS_CONFIG_DIR}:/config:ro
+          - "/data:/data"
+        depends_on:
+          - "sisock-crossbar"
+
+      ocs-influx-publisher:
+        image: simonsobs/ocs-influxdb-publisher-agent:latest
+        hostname: ocs-docker
+        volumes:
+          - ${OCS_CONFIG_DIR}:/config:ro
+
+      ocs-LSA99ZZ:
+        image: simonsobs/ocs-lakeshore372-agent:latest
+        hostname: grumpy-docker
+        network_mode: "host"
+        volumes:
+          - ${OCS_CONFIG_DIR}:/config:ro
+        command:
+          - "--instance-id=LSA99ZZ"
+          - "--site-hub=ws://10.10.10.2:8001/ws"
+          - "--site-http=http://10.10.10.2:8001/call"
+
+
+.. warning::
+
+    Bind mounts are a system unique property. This is especially true for ones
+    which use absolute paths. If they exist in any reference configuration
+    file, they will need to be updated for your system.
+
+Understanding what is going on in this configuration file is key to getting a
+system that is working smoothly. The Docker Compose reference_ explains the
+format of the file, for details on syntax you are encouraged to check the
+official documentation.
+
+In the remainder of this section we will go over our example. The first line
+defines the version of the docker-compose file format, which corresponds to the
+Docker Engine version you are running. You likely do not have to change this.
+
+Every block below ``services:`` defines a Docker container. Let's look at one
+example container configuration. This example does not represent something we
+would want to actually use, but contains configuration lines relevant to many
+other container configurations::
+
+  example-container-name:
+    image: simonsobs/example-docker-image:latest
```

Comment:
Maybe we should have a note or a warning about using the latest tag instead of an actual version number

Reply:
Great feedback -- added a warning about docker tags with a brief description of how to find them.